### DISCUSSION TITLE
Unable to append while using fieldArray and Controller

### DISCUSSION THREAD

**Discussion by gauravkrp** [react-hook-form/react-hook-form#6422] [Q&A] [answered]
### Version Number

7.14.2

### Codesandbox/Expo snack

https://codesandbox.io/s/optimistic-tereshkova-vrd3q?file=/pages/index.js

### Steps to reproduce

1. Go to CSB link https://codesandbox.io/s/optimistic-tereshkova-vrd3q?file=/pages/index.js
2. Click on 'Add Contact Number' button
3. See error

### Expected behaviour

While using native inputs, this was working fine. It started troubling only when I used `Controller `for `react-phone-input-2`

**Expected**: on clicking Add New `button` it should add/append a new field.

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
Unhandled Runtime Error
TypeError: Cannot read property '_f' of undefined
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by gauravkrp** [comment]
@bluebill1049 I see this issue moved here. Can you please investigate? 🙏 
I've trying it for so long but stuck in using fieldArray.

**Comment by bluebill1049** [comment, marked as answer]
this issue is related to the dev tool, which we already have an issue tracking there: https://codesandbox.io/s/polished-forest-lz2bg?file=/src/App.js https://github.com/react-hook-form/devtools/issues/103

**Reply by gauravkrp** [reply]
Oh! Thanks @bluebill1049
